integration/cmd_config_test.sh

```diff
@@ -42,6 +42,7 @@ all_body() {
     cat >"${HOME}/.kyua/kyua.conf" <<EOF
 syntax(2)
 architecture = "my-architecture"
+execenv = "my-env1 my-env2"
```

Comment:
The `execenv = ""` test case is missing.

Reply:
My time budget strategy was to only fix existing tests and work on additional tests when the design is finalized and agreed, what could go as the subsequent patch(es). And not to inflate the initial patch. Do you accept such approach?